### PR TITLE
feat: added badge component

### PR DESCRIPTION
**Pull request by Prashantrajput11**
added badge component : badge on avatar , badge on icons

image

image

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	position: relative;
+}
+.badge-status {
+	height: 2.2rem;
+	width: 2.2rem;
+	border-radius: 50%;
+	position: absolute;
+	margin-top: -57px;
+	margin-left: 100px;
+	border: 2px solid white;
```
Comment (manojasarsa) [inline components/badge.css]: you can make use of color variable

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.badge--status-active {
+	background-color: #00ff00;
+}
+
+.badge--status-offline {
+	background-color: #ff0000;
```
Comment (manojasarsa) [inline components/badge.css]: Here as well, use colors as variables

File: components/badge.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+.badge--status-away {
+	background-color: #313531ea;
+}
+
+/*******badge icon css****/
+
+.badge-icons-container {
+	display: flex;
+	justify-content: center;
```
Comment (manojasarsa) [inline components/badge.css:43]: Define, display flex and its properties as an utility class and use it across everywhere

File: components/badge.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.badge-icons {
+	/* display: block; */
+	width: 100px;
+	height: 100px;
+	/* border: 2px solid red; */
```
Comment (manojasarsa) [inline components/badge.css]: Remove comments if not needed

**Review by manojasarsa — APPROVED** [review]
Rest looks good!

File: components/badge.css (hunk 1 above)
Comment (Prashantrajput11) [reply components/badge.css]: fixed

File: components/badge.css (hunk 2 above)
Comment (Prashantrajput11) [reply components/badge.css]: fixed

File: components/badge.css (hunk 4 above)
Comment (Prashantrajput11) [reply components/badge.css]: fixed and removed

**Pull request merged**